packages/driver/cypress/integration/e2e/multidomain_spec.ts

```diff
@@ -180,5 +180,28 @@ describe('multidomain', { experimentalSessionSupport: true }, () => {
         })
       })
     })
+
+    it('allows users to call the "done" callback within the "switchToDomain" context synchronously', (done) => {
+    // @ts-ignore
+      cy.switchToDomain('foobar.com', () => {
+        cy
+        .get('[data-cy="cypress-check"]')
```

Comment:
throwing some commands onto the command queue to make sure they are executed before the done callback is invoked

Reply:
I also do not think this is desirable, where as we might want to clear out the command queue in the secondary domain as soon as done is called

Reply:
confirmed with @chrisbreiding  that immediately calling `done` should not wait for the command queue to finish. This should be fixed by the time this PR is ready for review